### PR TITLE
Update to latest bevy main

### PR DESCRIPTION
**Pull request by mcobzarenco**
Updates library to work with the latest bevy main

### CODE REVIEW

File: src/primitives.rs [hunk 1]
```diff
@@ -113,13 +116,7 @@ pub mod rays {
...
         ) -> Option<Self> {
             let view = camera_transform.compute_matrix();
-            let window = match windows.get(camera.window) {
-                Some(window) => window,
-                None => {
-                    error!("WindowId {} does not exist", camera.window);
-                    return None;
-                }
-            };
+            let window = get_window_for_camera(&windows, camera)?;
```
Comment (mcobzarenco) [inline src/primitives.rs:119]: `camera.window` was replaced by `camera.target`; this is an enum `RenderTarget` -- either a window or a texture:
```rust
pub enum RenderTarget {
    /// Window to which the camera's view is rendered.
    Window(WindowId),
    /// Image to which the camera's view is rendered.
    Image(Handle<Image>),
}
```

I refactored the match expressions that 1. either extract the window or 2. log an error and return `None` to a utility function, see below.

File: src/lib.rs [hunk 2]
```diff
@@ -445,7 +445,7 @@ pub fn ray_intersection_over_mesh(
         None => panic!("Mesh does not contain vertex positions"),
         Some(vertex_values) => match &vertex_values {
             VertexAttributeValues::Float32x3(positions) => positions,
-            _ => panic!("Unexpected types in {}", Mesh::ATTRIBUTE_POSITION),
+            _ => panic!("Unexpected types in {:?}", Mesh::ATTRIBUTE_POSITION),
```
Comment (mcobzarenco) [inline src/lib.rs:448]: The new `MeshVertexAttribute` doesn't impl `Display`

**Comment by aevyrie** [comment]
@mcobzarenco Thank you! Looks like this just needs a clippy fix and we can merge.

**Comment by mcobzarenco** [comment]
@aevyrie Ooops, sorry about that, should have checked -- I've fixed it and pushed force

**Comment by aevyrie** [comment]
No worries. Thanks!

**Pull request merged**